lib/public/SystemTag/ISystemTagManager.php

```diff
@@ -112,7 +112,7 @@ public function deleteTags($tagIds);
 	 *
 	 * @since 9.1.0
```

Comment:
```suggestion
	 * @since 9.1.0
	 * @since 31.0.0 `$user` can be null to check anonymous permissions
```

Also the typing above does not fit anymore, should be:
```
* @param IUser|null $user user to check permission for
```